lib/thread.h

```diff
@@ -240,6 +240,12 @@ extern void thread_call (struct thread *);
 extern unsigned long thread_timer_remain_second (struct thread *);
 extern struct timeval thread_timer_remain(struct thread*);
 extern int thread_should_yield (struct thread *);
+
+extern unsigned long timeval_elapsed (struct timeval a, struct timeval b);
```

Comment:
I understand, so I removed the timeval_copy function I created and replaced it by memcpy.
I can also move the timeval_add_msec in a similar manner.

BUT the timeval_elapsed function is an existing function which is still in thread.c for now (it has not been moved to monotime during the cleanup that was done).
 I'm just reusing this function which was available in the master branch.